docs/tutorials/genai_01_basic_tutorial.ipynb

```diff
@@ -30,7 +30,7 @@
   },
```

Comment:
<p>This should be formatted by Ruff</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/8651/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>